### PR TITLE
Meetup filter module

### PR DESCRIPTION
**Pull request by TruddyTheDuddi**
**What**
A fragment for the meetup list activity that allows to input filters

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -89,10 +89,10 @@
 
 
     <!-- Strings used for meetupList -->
-    <string name="show_all">Toutes les rencontres</string>
-    <string name="only_participating">pal prenant part</string>
-    <string name="only_creators">Créé par pal</string>
-    <string name="only_joined">rejoins</string>
+    <string name="meetup_filter_show_all">Toutes les rencontres</string>
+    <string name="meetup_filter_only_participating">pal prenant part</string>
+    <string name="meetup_only_creators">Créé par mes potes</string>
```
Comment (atappy) [inline app/src/main/res/values-fr/strings.xml:94]: maybe change to 'meetup_filter_only_creators' to keep it consistent

**Review by atappy — APPROVED** [review]
good idea to move the Calendar change to the viewModel, overall good job !

File: app/src/main/res/layout/fragment_criterions.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:text="@string/criterions_title"
-            android:fontFamily="@font/fredoka_bolder"
+            android:fontFamily="@font/fredoka_bold"
```
Comment (Gerussi) [inline app/src/main/res/layout/fragment_criterions.xml:23]: That's a ... bold move.

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -245,9 +245,18 @@
     <string name="followers_handle_title">Abonnés</string>
     <string name="profile_last_attended">Dernières participations</string>
     <string name="tags_no_tag">Pas d\'étiquettes</string>
+
+    <string name="meetup_filter_title">Filtrer les rencontres</string>
+    <string name="meetup_filter_subtitle">Selectionner une option pour réduir les résultats montrés</string>
+    <string name="meetup_filter_date_title">Selectionner des dates spécifiques</string>
```
Comment (Gerussi) [inline app/src/main/res/values-fr/strings.xml:251]: "Sélectionner" takes a "é"

File: app/src/main/res/layout/fragment_criterions.xml (hunk 2 above)
Comment (TruddyTheDuddi) [reply app/src/main/res/layout/fragment_criterions.xml:23]: 🥇

**Pull request merged**